content/03.results.md

```diff
@@ -117,57 +122,58 @@ Table: **Molecular subtypes generated through the OpenPBTA project.** Listed are
 
 ### Somatic Mutational Landscape of Pediatric Brain Tumors
 
-We performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,969 tumors (N = 1,019 RNA-Seq, N = 1,719 WGS, N = 64 WXS/Panel) and 38 cell lines (N = 16 RNA-Seq, N = 22 WGS) from 943 patients.
-Following SNV consensus calling (**Figure {@fig:S1}** and **Figure {@fig:S2}A-G**), we observed lower expected tumor mutation burden (TMB) **Figure {@fig:S2}H** in pediatric tumors compared to adult brain tumors from The Cancer Genome Atlas (TCGA), **Figure {@fig:S2}I**, with hypermutant (> 10 Mut/Mb) and ultra-hypermutant (> 100 Mut/Mb) tumors [@doi:10.1016/j.cell.2017.09.048] only found within HGGs.  
+We performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,043 tumors (N = 1,019 RNA-Seq, N = 918 WGS, N = 32 WXS/Panel) and 22 cell lines (N = 16 RNA-Seq, N = 22 WGS), from 943 patients, 833 with paired normal specimens (N = 801 WGS, N = 32 WXS/Panel). <!--SAMPLECOUNT-->
```

Comment:
1019+918+32 is 1969, but not finding the 1043 number you are getting.

Another note - `833 with paired normal specimens` is not technically correct. All of the tumors had paired normals (we did not include tumor only). BUT, if there were multiple phases of therapy, cell lines, etc, they all used that same normal. So this needs a reword. Perhaps just saying, 
>from 943 patients with paired normal specimens (N = 801 WGS, N = 32 WXS/Panel).

Reply:
1043 is the tumor count, not the biospecimen count. I don't think adding WGS + RNASeq makes much sense, as those are overlapping on the per-tumor level.

As to the paired specimens, here I am referring to the data we have listed in the histology file. We only have sequencing data for 833 patients. If all tumors have paired normal specimens, and there must be at least one normal per patient, then we should have some data for those listed somewhere, shouldn't we? I could rephrase as "833 with paired sequencing data", or if there is additional sequencing data for the missing 110 patient somewhere unlisted?

Reply:
>1043 is the tumor count, not the biospecimen count. I don't think adding WGS + RNASeq makes much sense, as those are overlapping on the per-tumor level.

Oh, gotcha. The way I was reading, I expected those numbers in parentheses to add up to the total N outside of parentheses. How about adding "biospecimens" to:

>We performed a comprehensive genomic analysis of somatic SNVs, CNVs, SVs, and fusions across 1,043 tumors (N = 1,019 RNA-Seq, N = 918 WGS, N = 32 WXS/Panel biospecimens)

>We only have sequencing data for 833 patients. 

Ahh, I think then the 943 comes from T/N + RNA-Seq in samples without T/N. I didn't think it was that many with RNA-only, but can check. That was confusing. So in that case, I would suggest:

"833 tumor DNA samples with paired normal DNA" ?

Reply:
I made it a bit wordier for the first instance, and removed the `N=`. I think the term "biospecimen" can be confusing in this context, so I wanted to avoid it.. I also added "paired normal DNA sequencing", which is at the patient, not tumor level.